Instructions in CONTRIBUTING.rst for running tests have an undocumented YAML dependency

I ran the following steps, based on instructions in CONTRIBUTING.rst:

```console
$ git clone git@github.com:PyconUK/ConferenceScheduler.git
$ cd ConferenceScheduler/

$ python3 -m venv env
$ source env/bin/activate
$ pip install -r requirements.txt

$ vi pytest.ini
$ cat pytest.ini
[pytest]
testpaths = tests docs
python_files =
    test_*.py
    *_test.py
    tests.py
pep8ignore =
    resources.py E701
addopts = --pep8 --doctest-glob='*.rst'

$ python setup.py test
zip_safe flag not set; analyzing archive contents...

Installed /Users/chana/repos/ConferenceScheduler/.eggs/pytest_runner-2.11.1-py3.6.egg
running pytest
running egg_info
creating src/conference_scheduler.egg-info
writing src/conference_scheduler.egg-info/PKG-INFO
writing dependency_links to src/conference_scheduler.egg-info/dependency_links.txt
writing requirements to src/conference_scheduler.egg-info/requires.txt
writing top-level names to src/conference_scheduler.egg-info/top_level.txt
writing manifest file 'src/conference_scheduler.egg-info/SOURCES.txt'
reading manifest file 'src/conference_scheduler.egg-info/SOURCES.txt'
writing manifest file 'src/conference_scheduler.egg-info/SOURCES.txt'
running build_ext
===================================================================== test session starts =====================================================================
platform darwin -- Python 3.6.1, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /Users/chana/repos/ConferenceScheduler, inifile: pytest.ini
plugins: pep8-1.0.6
collected 91 items

tests/conftest.py .
tests/test_resources.py .............
tests/test_scheduler.py ..........................
tests/test_validator.py ..............
tests/lp_problem/test_constraints.py .................
tests/lp_problem/test_objective_functions.py ....
tests/lp_problem/test_utils.py ..........
docs/conf.py .
docs/howto/define_a_conference.rst F
docs/howto/modify_tags_and_unavailability.rst .
docs/howto/obtain_mathematical_representation.rst .
docs/howto/use_different_solver.rst s
docs/tutorial/index.rst .

========================================================================== FAILURES ===========================================================================
______________________________________________________________ [doctest] define_a_conference.rst ______________________________________________________________
186                                                             'plenary',
187                                                             'workshop']}}
188
189 We can load the YAML document containing the 'session times' information in a
190 similar fashion. Again, the data is loaded into a Python dictionary with each
191 event type as a key mapping to a further dictionary with the session name as
192 key and a list of slot times as its values. The start times are converted to an
193 integer representing the number of seconds since midnight::
194
195     >>> import yaml
UNEXPECTED EXCEPTION: ModuleNotFoundError("No module named 'yaml'",)
Traceback (most recent call last):

  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest define_a_conference.rst[13]>", line 1, in <module>

ModuleNotFoundError: No module named 'yaml'

/Users/chana/repos/ConferenceScheduler/docs/howto/define_a_conference.rst:195: UnexpectedException
======================================================= 1 failed, 89 passed, 1 skipped in 1.96 seconds ========================================================
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,3 +9,4 @@ pyparsing==2.2.0
 pytest==3.0.7
 pytest-cache==1.0
 pytest-pep8==1.0.6
+PyYAML==3.12

